### ISSUE TITLE
Some models broken

### ISSUE THREAD

**Issue by lewa-j** [rein4ce/hlbsp#2]
Browser: Firefox
OS: Windows 10 1909
GPU: Radeon Vega 8
![img](https://user-images.githubusercontent.com/5416122/80305593-4b034000-87c6-11ea-8b10-a59dbc92bfc2.png)
Log: 
![console](https://user-images.githubusercontent.com/5416122/80305623-856cdd00-87c6-11ea-9bd4-7933d1731346.png)

**Comment by rein4ce** [comment]
Yeah, unfortunately that's the case. I think the model loading code doesn't always do the skinning properly but I never took the time to really investigate what's wrong.

**Comment by lewa-j** [comment]
I had a look right after a posted that issue. Cause is thedecoding of animation sequences . Original code was relied heavily on C pointers and wasn't properly adapted for js.
I managed to fix glitch on zombie, but simultaniously broke barney, so i didn't made pr.
